CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ set(PACKAGE_VERSION ${CMAKE_PROJECT_VERSION})
 set(LIBCHEWING_BINARY_VERSION 1.0.0)
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
+option(BUILD_TESTS "Build and run tests." ON)
+
+if(BUILD_TESTS)
+    enable_testing()
+endif()
+
 enable_testing()
```

Comment:
Generally it's advisable* to run the tests after the package build step to ensure it works with the compiler, compiler flags, system library, etc that are unique to the environment.

* Both Debian and Fedora recommend run checks after build.

The system only builds the test but does not run them by default, I wonder why we should disable ctest?

Reply:
for us (homerbew), we already have formula test, running build_tests, dont quite make sense (and we did disable whenever the option is avialable)